tv_apps/smart-system/js/preview_window.js

```diff
@@ -119,17 +119,21 @@
   };
 
   PreviewWindow.prototype._handle__opened = function(evt) {
-    var showPreviewHint = function() {
-      window.interactiveNotifications.showNotification(
-        window.InteractiveNotifications.TYPE.NORMAL, {
-          title: {
-            id: 'preview-app-hint'
-          },
-          text: {
-            id: 'add-to-apps'
-          },
-          icon: ADD_TO_APPS_ICON_PATH
-        });
+    var showPreviewHint = () => {
+      this.iframe.addEventListener('load', function _showPreviewHint() {
```

Comment:
If you want to handle the `showPreviewHint` at the "loaded" stage, what not listen to `_loaded` event [1] directly? I'm afraid there is a chance that "loaded" happens before "opened".

BTW, don't forget to remove the handler if you leverage `_loaded` event since "mozbrowserloadend" may be triggered multiple times.

[1] https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/js/app_window.js#L944